CMakeLists.txt

```diff
@@ -14,42 +14,33 @@ if(POLICY CMP0074)
   cmake_policy(SET CMP0074 NEW)
 endif()
 
-# we include this first to obtain version numbers from version.cmake
+# Version numbers are maintained in config/version.cmake
 include(config/version.cmake)
 
-#set up our project
 project(Imath VERSION ${IMATH_VERSION} LANGUAGES C CXX)
 
-#######################################
-#######################################
-# This declares all the configuration variables visible
+# ImathSetup.cmake declares all the configuration variables visible
 # in cmake-gui or similar and the rest of the global
-# project setup
-#
-# Please look at this file to see what is configurable
-#######################################
-#######################################
+# project setup. Check the context to see what is configurable
 include(config/ImathSetup.cmake)
 
-# generates config headers, package config files
+# Config headers and package config files
 add_subdirectory(config)
 
-# utility function for the repeated boilerplate of defining
-# the libraries
+# Utility function for the repeated boilerplate of defining the libraries
 include(config/LibraryDefine.cmake)
 
-#######################################
-
-# Include these modules without enable/disable options
+# Source code is in src/Imath
 add_subdirectory(src/Imath)
 
-# Tell CMake where to find the ImathConfig.cmake file. Makes it posible to call 
-# find_package(Imath) in downstream projects
+# Tell CMake where to find the ImathConfig.cmake file.  This makes it
+# posible to call find_package(Imath) in downstream projects
```

Comment:
Is it clear enough where the `this` points to? Logically I'd say, it's meaning is for the `ImathConfig.cmake` file, because that's the file used by downstream projects. But as a non-native English speaker, I got slightly confused whether it might belong to the line below the comment. Maybe for native speakers this is unambiguous and it's just due to lack of proper English skills.

Reply:
Thanks for that feedback. I rephrased it to be more clear, although if any cmake experts can clarify further, please do.